hal/src/samd21/usb/bus.rs

```diff
@@ -15,6 +19,12 @@ use usb_device::bus::PollResult;
 use usb_device::endpoint::{EndpointAddress, EndpointType};
 use usb_device::{Result as UsbResult, UsbDirection, UsbError};
 
+use crate::dbgprint;
+#[cfg(feature = "use_uart_debug")]
+use crate::uart_debug;
```

Comment:
I thought you were going to remove this?

Reply:
Ah yes, nice catch.